WordPress/Classes/Networking/AccountSettingsRemote.swift

```diff
@@ -16,13 +16,13 @@ class AccountSettingsRemote: ServiceRemoteREST {
         //
         // However it might be possible that the account gets deallocated and
         // when it's fetched again it would create a different api object.
-        let key = api.authToken.hashValue
+        let key = api.hashValue
```

Comment:
Just leaving a note, this is probably fine for this PR, but since this is `Hashable` now, we should consider using `api` directly as the key instead of its `hashValue`.